packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -307,6 +476,45 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
 		);
 	});
 
+	it("should not call entry point before data store is requested", async () => {
+		const dataStoreFactory = new TestFluidObjectFactory();
+		const dataStoreContext = new TestDataObjectContext();
+
+		assert.doesNotThrow(
+			async () => dataStoreFactory.instantiateDataStore(dataStoreContext, false),
```

Comment:
What is this validating? From the PR description, the validation should be that finishInitialization is not called which exists on a `DataObject`. So, I would expect the test to validate that a `DataObject`'s finishInitialization is not called.

Reply:
Here is verifying the entryPoint function won't be called when the data store just initiates.